modules/Microsoft.VirtualMachineImages/imageTemplates/deploy.bicep

```
@@ -147,11 +164,13 @@ resource imageTemplate 'Microsoft.VirtualMachineImages/imageTemplates@2020-02-14
     vmProfile: {
       vmSize: vmSize
       osDiskSizeGB: osDiskSizeGB
+      userAssignedIdentities: !empty(vmUserAssignedIdentities) ? vmUserAssignedIdentities : []
       vnetConfig: !empty(subnetId) ? vnetConfig : null
     }
     source: imageSource
     customize: customizationSteps
     distribute: distribute
+    stagingResourceGroup: !empty(stagingResourceGroup) ? stagingResourceGroup : ''
```

The default value is already an empty string, so the condition is not needed
```suggestion
    stagingResourceGroup: stagingResourceGroup
```

---

fixed